lib/google/cloud/gemserver.rb

```diff
@@ -22,13 +22,25 @@ module Cloud
     # a gemserver to a Google Cloud Platform project.
     #
     module Gemserver
-      autoload :CLI,           "google/cloud/gemserver/cli"
-      autoload :VERSION,       "google/cloud/gemserver/version"
-      autoload :Configuration, "google/cloud/gemserver/configuration"
-      autoload :StorageSync,   "google/cloud/gemserver/storage_sync"
-      autoload :GCS,           "google/cloud/gemserver/gcs"
-      autoload :Authentication,"google/cloud/gemserver/authentication"
-      autoload :Backend,       "google/cloud/gemserver/backend"
+      autoload :CLI,            "google/cloud/gemserver/cli"
+      autoload :VERSION,        "google/cloud/gemserver/version"
+      autoload :Configuration,  "google/cloud/gemserver/configuration"
+      autoload :StorageSync,    "google/cloud/gemserver/storage_sync"
```

Comment:
Since the `StorageSync` class is only run on App Engine it can be wrapped by `Backend`.